packages/yoastseo/src/scoring/assessments/seo/KeyphraseDistributionAssessment.js

```diff
@@ -66,7 +65,7 @@ class KeyphraseDistributionAssessment extends Assessment {
 
 		assessmentResult.setScore( calculatedResult.score );
 		assessmentResult.setText( calculatedResult.resultText );
-		assessmentResult.setHasMarks( this._keyphraseDistribution.sentencesToHighlight.length > 0 );
+		assessmentResult.setHasMarks( calculatedResult.hasMark );
```

Comment:
It is not crucial, and I also don't have a better idea. But it is a bit bothering that we are using plural (`hasMarks`) and singular (`hasMark`). But very minor, so feel free to ignore.

Reply:
It's a really good point! very sharp 👀  
I'll change this